12_glob_lab/globlab.txt

```diff
@@ -1,29 +1,59 @@
 Glob Lab.
 https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
-1. Type in a glob pattern to match:
+1. 
+
+Type in a glob pattern to match:
 
-  1. All files whose name starts with a lowercase letter
+ 
+ 1. All files whose name starts with a lowercase letter
   Answer: 
 
-  2. All files whose name starts with an uppercase letter
+ 
+ls -d [[:lower:]]*/*
+ 2. All files whose name starts with an uppercase letter
   Answer:
 
-  3. All files whose name ends in an "a", "e", "i", "o" or "u"
+ 
+ls -d [[:upper:]]*/* - for some reason shows the following:
+Doc/README.txt Doc/test.c
+Why test.c as well?
+
+ 3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
-  4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
+ 
+echo *[a,e,i,o,u]
+
+ 4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
 
-  5. All files with extension ".c" that are in a folder named "src"
+ 
+echo *[!aeiou]
+
+ 5. All files with extension ".c" that are in a folder named "src"
   Answer:
 
-  6. All files with extension ".c" under any folder
+
+
+ls *.c src/
+  
+6. All files with extension ".c" under any folder
   Answer:
 
-2. Write a command that would delete all files ending with a ".png" under any subdirectory.
+
+find -name '*.c'

```

Comment:
Command should be `ls */*.png` (we'll talk about the reasons tomorrow if it's still not clear)